airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -24,6 +24,7 @@
 from botocore.exceptions import ClientError, WaiterError
 
 from airflow import AirflowException
+from airflow.exceptions import AirflowProviderDeprecationWarning
```

Comment:
```suggestion
from airflow.exceptions import AirflowException, AirflowProviderDeprecationWarning
```

Drive-by improvements.